qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/commons/ResolvedJavaTypeInfo.java

```diff
@@ -247,4 +249,14 @@ private static boolean isEmpty(String value) {
 		return value == null || value.isEmpty();
 	}
 
+	@Override
+	public String toString() {
+		ToStringBuilder b = new ToStringBuilder(this);
+		b.add("name", this.getName());

```

Comment:
Also could define utility class/constants for Java type info

Reply:
fixed,  but I have not created a constant class for that, we will see of we will need it in the future.